codebreaker_garage/spec/codebreaker_garage/console_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+module Codebreaker_garage
+  RSpec.describe Console do
+    context '#start' do
+      before do
+        allow(subject.instance_variable_get(:@game)).to receive(:any_attempts?).and_return(false)
+      end
+
+      describe 'printing messages' do
+        it 'says that new game has been started' do
+          allow(subject).to receive(:gets).and_return('exit')
+          expect { subject.start }.to output(/New game has been started./).to_stdout
+        end
+
+        it "shows hint if user asks" do
+          allow(subject).to receive(:gets).and_return('hint')
```

Comment:
I can't figure out what is going on. `allow(subject).to receive(:gets).and_return('....')` freezes the terminal and nothing happens.